install.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+#
+# This is a simple installer script for KubeOne 
+#
+#
+# What OS is used
+OS=$(uname)
```

Comment:
As mentioned for the README, this doesn't work on Linux. I like the script approach, but we should make it work on Linux as well.

Reply:
It should be possible to use `tr` or `awk` to lowercase the string:
```
OS=$(uname | tr '[:upper:]' '[:lower:]')
```
or with `awk`
```
OS=$(uname | awk '{print tolower($0)}')
```
(reference: https://stackoverflow.com/questions/2264428/how-to-convert-a-string-to-lower-case-in-bash)